tests/testthat/test-gadgets.R

```diff
@@ -35,17 +55,11 @@ test_that("makeArrayGadget passes shiny tests", {
     skip_on_travis()
     skip_on_cran()
     test_output <- shinytest::testApp("gadgets/makeArrayGadget", compareImages = FALSE)
-    print(test_output)
     shinytest::expect_pass(
         shinytest::testApp("gadgets/makeArrayGadget", compareImages = FALSE, quiet = TRUE))
 })
 
 with_mock_crunch({
-    test_that("listDatasets gets expected input from other Crunch functions", {
-        expect_is(names(projects()), "character")
-        expect_is(listDatasets(), "character")
-    })
```

Comment:
Are these tests duplicated somewhere else? They seem a little on the silly side, but I'm sure they are here for a reason, no?

Reply:
Makes sense to me to delete them since they're not testing functionality from this package.